os/binfmt/elf.c

```diff
@@ -74,11 +74,11 @@
  * Pre-processor Definitions
  ****************************************************************************/
 
-/* CONFIG_DEBUG, CONFIG_DEBUG_INFO, and CONFIG_DEBUG_BINFMT have to be
+/* CONFIG_DEBUG, CONFIG_DEBUG_VERBOSE, and CONFIG_DEBUG_BINFMT have to be
  * defined or CONFIG_ELF_DUMPBUFFER does nothing.
  */
 
-#if !defined(CONFIG_DEBUG_INFO) || !defined(CONFIG_DEBUG_BINFMT)
+#if !defined(CONFIG_DEBUG_VERBOSE) || !defined(CONFIG_DEBUG_BINFMT)
```

Comment:
Use CONFIG_DEBUG_BINFMT_INFO stand alone is not enough?

Reply:
done